qiskit/extensions/quantum_initializer/diagonal.py

```diff
@@ -30,6 +30,7 @@
 from qiskit.circuit import Gate
 from qiskit.circuit.quantumcircuit import QuantumCircuit, QuantumRegister
 from qiskit.exceptions import QiskitError
+from qiskit.quantum_info.operators.predicates import ATOL_DEFAULT, RTOL_DEFAULT
 
 _EPS = 1e-10  # global variable used to chop very small numbers to zero
```

Comment:
This can be removed now, right?

Reply:
This is used below to make sure both UnitaryGate and DiagonalGate are in agreement.

Reply:
https://github.com/Qiskit/qiskit-terra/blob/69c0cb399c18f15e081812058680120a024568a2/qiskit/extensions/quantum_initializer/diagonal.py#L64

Reply:
Sorry, I meant I think the `_EPS` is unused.

Reply:
Sorry, I thought this referenced ATOL_DEFAULT. Removed.